binlog_streamer.go

```diff
@@ -165,6 +165,10 @@ func (s *BinlogStreamer) Run() {
 			// This event can tell us about table structure change which means
 			// the cached schemas of the tables would be invalidated.
 			// TODO: investigate using this to allow for migrations to occur.
+		case *replication.GenericEvent:
+			// go-mysql don't parse this event, it use GenericEvent instead,
```

Comment:
"go-mysql don't parse all events and unparsed events are denoted with empty GenericEvent structs"